steward/src/commands.rs

```diff
@@ -78,13 +78,21 @@ impl Configurable<StewardConfig> for EntryPoint {
             .map(PathBuf::from)
             .unwrap_or_else(|| CONFIG_FILE.into());
 
-        if filename.exists() {
+        let config_env_variable = std::env::var("VARIABLE").is_ok();
+
+        if config_env_variable {
```

Comment:
We'll have to update the logic of our overall check here. Before we get to this `if` statement, we should be recording the configured `filename` and the `config_env_variable`. The current code checks for a defined path, and if it doesn't find one, it returns `CONFIG_FILE` as the file path.

Our expected logic is:
1. use the config path if it's provided
2. use the environment variable if it's provided
3. try the default CONFIG path
4. return None

This means that we can't `unwrap_or_else(|| CONFIG_FILE.into()` any more, because that makes cases 1 and 3 happen at the same time, so we'd have to leave it as an `Option` until the `if` check.

Then, we'd have to check
1. `filename.is_some()` and if so, use it
2. `config_env_variable.is_ok()` and if so, use it
3. construct a PathBuf from `CONFIG_FILE` and if it exists, use it
4. return `None`